backend/auth.py

```diff
@@ -64,7 +73,9 @@ def wrapper(*args, **kwargs):
                         issuer=DOMAIN,
                         audience='77ee33cd-cc7f-4b7a-bce9-241c96458f14',
                         options={"verify_signature": verify,
-                                 "verify_exp": verify}
+                                 "verify_exp": verify,
+                                 "verify_at_hash": verify_at_hash},
```

Comment:
We should check this

Reply:
I've created try/except blocks to respond properly if either the ID token or the access token is missing. The ID token is always attempted first, so if both are missing that one will be commented on.